homeassistant/components/androidtv/media_player.py

```diff
@@ -356,7 +358,7 @@ def _adb_decorator(func):
         @functools.wraps(func)
         async def _adb_exception_catcher(self, *args, **kwargs):
             """Call an ADB-related method and catch exceptions."""
-            if not self.available and not override_available:
+            if not self._attr_available and not override_available:
```

Comment:
As discussed in other PRs, let's not read the `self_attr_available` directly, use the `self.available` property function instead